src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -195,13 +195,11 @@ private function createAuthorization($config, ContainerBuilder $container)
         ));
 
         foreach ($config['access_control'] as $access) {
-            $matcher = $this->createRequestMatcher(
-                $container,
-                $access['path'],
-                $access['host'],
-                $access['methods'],
-                $access['ips']
-            );
+            if (null !== $access['matcher']) {
+                $matcher = new Reference($access['matcher']);
```

Comment:
the currently created matcher is configured by many props in `$access`. Thus, is this extension point really enough when you don't give it these parameters?